reasoning.md

```diff
@@ -24,3 +24,15 @@ We believe active membership is important as it brings a host of other benefits
 Active membership entails voting (democracy) and meetings as a base, but importantly involves recognized volunteering work that helps the consortium function and thus spreads the administrative work to all. An important reason this should be recognized is that it brings about transparency that motivates others to chip in as well. A financial stake of recurring payment is also a form of active membership, and in order for it not to be excluding, as the financial situation can vary a lot between the different members, the amount is kept flexible. Lastly, face-to-face meetings are an important part of being an active member as there is little substitute for physical gatherings.
 
 One important side-effect of being an active member is that it fosters trust through active collaboration. Both for the parties involved in the collaboration but also for other members trusting in a good outcome. Furthermore, face-to-face gatherings over a weekend allows members to see each others as humans, laying the foundation for a culture of care. When nurtured this should expand into positive feedback loops that strengthens the consortium.
+
+## Handling Conflicts
+
+As much as possible, we try to view conflicts as opportunities for growth, both in members and for the consortium: members have an opportunity to better understand the values and experiences they bring with them; the consortium gets an opportunity to revise its practices and documents. We therefore encourage members involved in a conflict to reflect and articulate their underlying personal needs and previous experiences that support their viewpoint. A clear understanding of both needs and past experiences, from all sides of a conflict, can make it easier to resolve, with, for example, voluntary behaviour changes from parties or revised practices in the consortium. This in turn leads to an increase in the level of trust and collaboration betwen members. This is the best case and preferred scenario.
+
```

Comment:
I feel here on line 31 (i.e. between these two paragraphs) is where a mediated conflict resolution process would sit, but maybe it's like Zenna said, maybe too early & not needed yet to define that.

Reply:
I did not have a good process to propose yet. Currently, I am thinking that if two or more people are in conflict, the responsibility for mediating would best fall on the people that vouch for them (ex: at least one for each person in a conflict). 

That way we don't have to elect a community manager/leader and distribute the responsibility in the consortium. This would also give actual weight to vouching someone else because it comes with the responsibility of being available to deal with potential conflicts they may find themselves in. Seen in the reverse direction, if someone is not willing to engage on conflicts the people they brought in may potentially generate, perhaps the vouching was not strong enough in the first place.